### PR TITLE
2: text to css

### PR DESCRIPTION
**Pull request by chrisnojima**
This switches text to use css to increase perf

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

**Comment by chrisnojima** [comment]
cc: @cecileboucheron @adamjspooner to sanity check

**Comment by cecileboucheron** [comment]
Shouldn't we try to use our globalColors constants instead of hard-coded rgb values?

**Comment by chrisnojima** [comment]
this is a real css file so i can't pull in those vars. we can make the pipeline more complicated later i guess but it didn't seem worth it right now

**Comment by cecileboucheron** [comment]
Ok. We'll have to remember to change this file if we change our colors.

**Comment by chrisnojima** [comment]
sure, also the text colors change very very rarely

File: shared/common-adapters/text.css [hunk 1]
```diff
@@ -0,0 +1,295 @@
+/* Used by text on desktop to speed up perf */
```
Comment (buoyad) [inline shared/common-adapters/text.css]: I found a text type that's missing here (`BodyTinyBold`). Do we have a way to make sure this doesn't diverge with `text.meta.*`?

Comment (chrisnojima) [reply shared/common-adapters/text.css]: longer term we could generate this , but i'm keeping this simple for now

**Comment by buoyad** [comment]
There are some things that use text's `getStyle` (mostly input components afaict), so they're only getting partial styles on this branch

File: shared/common-adapters/text.css (hunk 1 above)
Comment (chrisnojima) [reply shared/common-adapters/text.css]: added it back in

**Comment by chrisnojima** [comment]
@buoyad i added back getStyle but its only called by external components, not text. we should reconcile that better soon

**Comment by MarcoPolo** [comment]
What about generating these at load like icons? Not super important but might avoid some of the issues mentioned

**Comment by chrisnojima** [comment]
Yeah lets ticjet  this

File: shared/common-adapters/text.desktop.js [hunk 2]
```diff
@@ -62,31 +62,70 @@ class Text extends Component<Props> {
...
+  const textDecoration = meta.isLink && backgroundMode !== 'Normal' ? {textDecoration: 'underline'} : null
+
+  return {
+    ...colorStyle,
+    ...lineClampStyle,
+    ...clickableStyle,
+    ...selectableStyle,
+    ...textDecoration,
+  }
+}
```
Comment (MarcoPolo) [inline shared/common-adapters/text.desktop.js]: Couldn't these also be handled by css?

**Pull request merged**